src/dmd/expressionsem.d

```diff
@@ -728,7 +728,6 @@ Expression resolvePropertiesOnly(Scope* sc, Expression e1)
 
     Expression handleOverloadSet(OverloadSet os)
     {
-        assert(os);
```

Comment:
Make it an in contract?

Reply:
will they be compile when -release will be removed ? The problem here (and the next two assert) is that since the parameter is used right after you got a crash so asserting has no value here. itt's not like a crash would happen 200 lines after.

Reply:
No, but assertions are more informative (though that's just my opinion and not fact).